srcpkgs/nsjail/template

```diff
@@ -11,7 +11,7 @@ license="Apache-2.0"
 homepage="http://nsjail.com"
 distfiles="https://github.com/google/nsjail/archive/${version}.tar.gz
 https://github.com/google/kafel/archive/722b93a630e53ca472a4c69f61cb3dfc44ea5041.tar.gz"
-checksum="f8578a48330d3d089c9234a2b562be08043f8fe77453c322724acdf403d0afc3
+checksum="cfa66d3ed136b2e221752287b95e544915e8a6760aa866f023b604d14a374919
  49d2327c794c90aca50a9933c00d453705534b16e5f42d8c8e19c3d274f5e5f8"
```

Comment:
The commit for kafel was changed as well, you should update it: https://github.com/google/nsjail/tree/3.0